masonry/src/text/layout.rs

```diff
@@ -501,6 +516,89 @@ impl<T: AsRef<str> + Eq> TextLayout<T> {
             &self.layout,
         );
     }
+
+    pub fn accessibility(&mut self, ctx: &mut AccessCtx, parent_node: &mut NodeBuilder) {
```

Comment:
As you predicted, this using `AccessCtx` is a slight layering break. I think for expedience this is fine, although I think if we can pass in the individual fields (I think only `tree_update` is used?)